src/gui/addnewtorrentdialog.cpp

```diff
@@ -755,6 +761,25 @@ void AddNewTorrentDialog::displayContentTreeMenu(const QPoint &)
     menu->popup(QCursor::pos());
 }
 
+void AddNewTorrentDialog::displayColumnHeaderMenu(const QPoint &)
+{
+    QMenu *menu = new QMenu(this);
+    menu->setAttribute(Qt::WA_DeleteOnClose);
+    menu->setToolTipsVisible(true);
+
+    QAction *resizeAction = menu->addAction(tr("Resize columns"), this, [this]()
+    {
+        for (int i = 0; i < m_ui->contentTreeView->header()->count(); ++i)
+        {
+            if (!m_ui->contentTreeView->isColumnHidden(i))
+                m_ui->contentTreeView->resizeColumnToContents(i);
+        }
+    });
+    resizeAction->setToolTip(tr("Resize all non-hidden columns given to the size of its contents"));
```

Comment:
```suggestion
    resizeAction->setToolTip(tr("Resize all non-hidden columns to the size of their contents"));
```

I'm not a native english speaker and i could be wrong about "given" but it seems out of place.

Reply:
It was from [Qt doc](https://doc.qt.io/qt-5/qtreeview.html#resizeColumnToContents ) and it looked strange to me too.

Updated, thanks for the suggestion.